src-built-in/components/toolbar/stores/searchStore.js

```diff
@@ -86,7 +85,14 @@ var Actions = {
 			})
 		})
 	},
+
+	/**
+	 * Positions a dropdown window under the search bar containing the search results.
+	 * Returns immediately if the text is empty so a search results menu doesn't appear
+	 */
 	positionSearchResults() {
+		let text = document.getElementById("searchInput").innerHTML;
```

Comment:
Please refactor to avoid UI access within store functions.   I acknowledge that this preexisting, but it needs to be corrected rather than continued.